composer.json

```diff
@@ -135,7 +135,7 @@
         "dragonmantank/cron-expression": "^3.1",
         "egulias/email-validator": "^2.1.10|^3.1|^4",
         "guzzlehttp/promises": "^1.4|^2.0",
-        "jolicode/jolinotif": "^2.7.2",
+        "jolicode/jolinotif": "^2.7.3|^3.0",
```

Comment:
Why do we disallow 2.7.2 now?

Reply:
I'm used to bump dependency requirements AT some time, but that's maybe not adapted for the framework. I will revert this if not wanted 👍

Reply:
Unless we have a good reason for this bump, I wouldn't do it.